mlrun/runtimes/nuclio/function.py

```diff
@@ -599,6 +599,15 @@ def deploy(
         # when a function is deployed, we wait for it to be ready by default
         # this also means that the function object will be updated with the function status
         self._wait_for_function_deployment(db, verbose=verbose)
+        background_tasks = mlrun.common.schemas.BackgroundTaskList(
```

Comment:
just a suggestion to make it more clear
```suggestion
        # check if there are any background tasks related to creating model endpoints
        model_endpoints_creation_background_tasks = mlrun.common.schemas.BackgroundTaskList(
```

Reply:
Note that I also changed the variable name